src/components/Modal/utils.ts

```diff
@@ -5,25 +5,22 @@ export type ModalHook = {
   toggleModal: (e?: React.MouseEvent, open?: boolean) => boolean
 }
 
-export const useModal = (): ModalHook => {
-  const [isOpen, setIsOpen] = useState(false)
+export const useModal = (isInitiallyOpen: boolean | undefined): ModalHook => {
+  const [isOpen, setIsOpen] = useState(isInitiallyOpen || false)
 
   const allowToggle = (e: React.MouseEvent): boolean => {
     const clickedElement = e.target as Element
 
     if (e && clickedElement) {
       if (clickedElement.closest('.usa-modal')) {
         // Element is inside a modal
-        if (
+
+        // Element is a close button - proceed
+        // Don't allow opening a modal from within a modal
+        return (
           clickedElement.hasAttribute('[data-close-modal]') ||
-          clickedElement.closest('[data-close-modal]')
-        ) {
-          // Element is a close button - proceed
-          return true
-        } else {
-          // Don't allow opening a modal from within a modal
-          return false
-        }
+          !!clickedElement.closest('[data-close-modal]')
+        )
```

Comment:
Nice. Might want to update the comment to go with it. maybe `Only allow toggle if element is a close button`